.NET docs updates (#3540)
getsentry/sentry-docs: src/includes/getting-started-primer/dotnet.mdx

Our .NET SDK supports all recent versions of the platform, and integrates well with a variety of popular frameworks and packages in the ecosystem. It gives developers helpful hints for where and why an error or performance issue might have occurred.

> The main [Sentry NuGet package](https://www.nuget.org/packages/Sentry) targets .NET Standard 2.0. [Our legacy SDK](/clients/csharp/) supports .NET Framework as early as 3.5. Read more about [compatibility and unit testing](./unit-testing.mdx).

Reviewer: I think here we could say that:

> The lowest version this SDK is compatible with are .NET Standard 2.0 and .NET Framework 4.6.1.
> For older versions, such as .NET Framework 3.5, please refer to our Raven SDK.

The reference about Unit Testing here isn't a good fit I believe. Also something we can just move to configuration somehow.

**Features:**

> - <PlatformLink to="/enriching-events/breadcrumbs/#automatic-breadcrumbs">Breadcrumbs automatically</PlatformLink> captured

Reviewer: To really clear, is it worth pointing out something like:
```suggestion
- Multiple integrations with additional features such as Serilog, log4net, ASP.NET Core, EntityFramework, Xamarin.
- <PlatformLink to="/enriching-events/breadcrumbs/#automatic-breadcrumbs">Breadcrumbs automatically</PlatformLink> captured
```

?

- <PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink> enrich your event by storing additional files, such as config or log files.
- <PlatformLink to="/enriching-events/user-feedback/">User Feedback</PlatformLink> provides the ability to collect user information when an event occurs.
- [Performance Monitoring](/product/performance/) creates transactions to track the performance of your application

> - Samples provided for [C#](https://github.com/getsentry/sentry-dotnet/tree/main/samples), [F# Console](https://github.com/sentry-demos/fsharp), and[F# Giraffe](https://github.com/sentry-demos/giraffe)

Reviewer: ```suggestion
- Samples provided for [multiple C# app models](https://github.com/getsentry/sentry-dotnet/tree/main/samples), [F# Console](https://github.com/sentry-demos/fsharp), and[F# Giraffe](https://github.com/sentry-demos/giraffe)
```

Our .NET SDK supports all recent versions of the platform, and integrates well with a variety of popular frameworks and packages in the ecosystem. It gives developers helpful hints for where and why an error or performance issue might have occurred.

This <PlatformLink to="/compatibility/">SDK is compatible</PlatformLink> with .NET Standard 2.0 and .NET Framework 4.6.1. For older versions, such as .NET Framework 3.5, please refer to our [legacy SDK](/clients/csharp/).

**Features:**

- Multiple integrations with additional features such as Serilog, log4net, ASP.NET Core, EntityFramework, Xamarin.

> - <PlatformLink to="/enriching-events/breadcrumbs/#automatic-breadcrumbs">Breadcrumbs automatically</PlatformLink> captured

Reviewer: ```suggestion
- <PlatformLink to="/enriching-events/breadcrumbs/#automatic-breadcrumbs">Breadcrumbs automatically</PlatformLink> captured.
```

Author: actually, we don't want periods on bullets. I'll fix as I see this is inconsistent in the list.

- <PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink> enrich your event by storing additional files, such as config or log files.
- <PlatformLink to="/enriching-events/user-feedback/">User Feedback</PlatformLink> provides the ability to collect user information when an event occurs.
- [Performance Monitoring](/product/performance/) creates transactions to track the performance of your application
- Automatically captures errors including: unhandled exceptions and unobserved task exceptions.

> - Events [enriched](https://docs.sentry.io/platforms/apple/enriching-events/context/) with device data.

Reviewer: ```suggestion
- Events <PlatformLink to="/enriching-events/context/">enriched</PlatformLink>  with device data.
```
